index.html

```diff
@@ -153,78 +158,80 @@ <h3>Multi-Post Stories</h3>
+        </div>
+        <div id="about_me_description_container">
+          <h2 class="h2_gray_title">About myself</h2>
+          <p id="about_me_description">
+            Hello I’m a software developer! I can help you build a product ,
+            feature or website. Look through some of my work and experience! If
+            you like what you see and have a project you need coded, don’t
+            hesitate to contact me.
+          </p>
+          <p id="lets_connect">LET'S CONNECT</p>
+          <div id="socials">
+            <ul>
+              <li>
+                <img src="assets/socials/twitter.svg" alt="Twitter icon" />
+              </li>
+              <li>
+                <img src="assets/socials/linkedin.svg" alt="Linkedin icon" />
+              </li>
+              <li>
+                <img src="assets/socials/m.svg" alt="m icon" />
+              </li>
+              <li>
+                <img src="assets/socials/github.svg" alt="Github icon" />
+              </li>
+              <li>
+                <img src="assets/socials/victory.svg" alt="victory icon" />
+              </li>
+            </ul>
+          </div>
+          <button class="card_btn">Get my resume</button>
+        </div>
       </div>
-      <div id="about_me_description_container">
-        <h2 class="h2_gray_title">About myself</h2>
-        <p id="about_me_description">
-          Hello I’m a software developer! I can help you build a product ,
-          feature or website. Look through some of my work and experience! If
-          you like what you see and have a project you need coded, don’t
-          hesitate to contact me.
-        </p>
-        <p id="lets_connect">LET'S CONNECT</p>
-        <div id="socials">
-          <ul>
-            <li>
-              <img src="assets/socials/twitter.svg" alt="Twitter icon" />
-            </li>
-            <li>
-              <img src="assets/socials/linkedin.svg" alt="Linkedin icon" />
-            </li>
+
+      <div id="skills_container">
+        <ul class="dropdown">
+          <li class="dropdown-btn">
+            <p>Languages</p>
+            <img src="assets/arrow/arrow.svg" alt="" />
+          </li>
+          <ul class="dropdown-content">
             <li>
-              <img src="assets/socials/m.svg" alt="m icon" />
+              <img src="assets/languages/js.svg" alt="javascript" />
+              <p>JavaScript</p>
             </li>
             <li>
-              <img src="assets/socials/github.svg" alt="Github icon" />
+              <img src="assets/languages/html.svg" alt="html" />
+              <p>HTML</p>
             </li>
             <li>
-              <img src="assets/socials/victory.svg" alt="victory icon" />
+              <img src="assets/languages/css.svg" alt="css" />
+              <p>CSS</p>
             </li>
           </ul>
-        </div>
-        <button class="card_btn">Get my resume</button>
-      </div>
-    </section>
```

Comment:
As I mentioned above, kindly I suggest you update all your cards to stick to the design as much as possible `(e.g., font, colors, images, text, margins)` using [the info of the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).

> ## The Figma Design
>
><img width="742" alt="Screenshot 2023-06-08 at 1 41 11 PM" src="https://github.com/Mehr-o-maah/Portfolio_2/assets/91237525/04d665f1-a387-4478-9e40-2e45eed76d9d">
><img width="195" alt="Screenshot 2023-06-08 at 1 35 25 PM" src="https://github.com/Mehr-o-maah/Portfolio_2/assets/91237525/a8486a66-9a29-4842-9f3a-74565a209de6">
>
> ## Your Design
>
><img width="1399" alt="Screenshot 2023-06-08 at 1 03 56 PM" src="https://github.com/Mehr-o-maah/Portfolio_2/assets/91237525/1f0617f6-b638-4a4e-a2a2-6900253258d3">
><img width="374" alt="Screenshot 2023-06-08 at 1 18 41 PM" src="https://github.com/Mehr-o-maah/Portfolio_2/assets/91237525/5e65cff3-3466-4493-9f63-8f7405939fe3">